tests/test-rest-meta-users-controller.php

```diff
@@ -0,0 +1,1087 @@
+<?php
```

Comment:
Given the similarity between them, it would be nice to have a common abstraction for meta unit tests. It can happen in a follow-up PR though.